ddl/db_integration_test.go

```diff
@@ -3821,3 +3821,24 @@ func TestIssue29326(t *testing.T) {
 	err = tk.ExecToErr("create definer=`root`@`127.0.0.1` view v1 as select t1.id, t2.id from t1,t2 where t1.id=t2.id")
 	require.True(t, infoschema.ErrColumnExists.Equal(err))
 }
+
+func (s *testIntegrationSuite3) TestInvalidPartitionNameWhenCreateTable(c *C) {
```

Comment:
@mjonss You need to use new test framework to implement it. those test cases have been migrated.